packages/components/src/toolbar/toolbar-button-container.js

```diff
@@ -0,0 +1,8 @@
+export default ( props ) => (
+	<div
+		key={ props.keykeyProp }
```

Comment:
shouldn't this be `prop.keyProp` only

Reply:
My understanding of the docs is that `key` is used for the rendering reconciliation logic and since it was there in the original code (`div`) it needs to be here in the web variant of the component too. I can be wrong though as my React knowledge is limited still.

Reply:
but keykeyPro?

Reply:
Oh, I see now, that's a typo, good catch! Fixed with aed7db649.